text/052-platform-release-cycle.md

```diff
@@ -0,0 +1,158 @@
+of Coq with the release of other packages. It provides scripts that build
+that selection of Coq packages on Windows, Mac and Linux from sources
+on the user's machine, and as well binary installers (prebuilt) for users
+to quickly get a working environment.
+
+This product is what users will test in this new model; Coq is a core
+component of it.
+
+# Detailed design
+
+The new process' timeline:
+```
+                     D             D            D   D  D
+coq      --+---(1)---+------(2)----+-----(3)----+---+--+----
+ vX branch/          |             |            |
+            X+rc tag/              |            |
+                           X.0 tag/             |
+                                        X.1 tag/
+
+                                   I                    I
+platform  -----------+----(4)------+--------(5)---------+
+           vX branch/              |                    |
+                        X+beta tag/                     |
+                                                X.0 tag/
+Artifacts:
+- D = docker image for Coq (and opam package)
+- I = binary installers for Coq platform (and opam packages and docker image)
+```
+
+## Coq
+
+On time based schedule the RM branches vX.
+
+1. The RM shepherds the few PR which are ready and pins projects tracked by CI
+   (using commit hashes, not necessarily tags). Then he tags Coq. This should
+   take 10 days. No packages are built, just a git tag. A docker image is built,
+   so that project maintainers can use it in CI. No breaking change is allowed
+   from now on (unless a severe problem is found).
+2. Doc is updated (eg. Changes file) and eventual fixes required by the platform
+   are done. Ideally no other change is done. This should take 10 days.
+3. In response to a severe bug report Coq devs make an hotfix in master which is
+   backported to vX by the RM which then tags a point release, possibly as soon
+   as the fix is available and merged.
+
+## Platform
+
+When Coq X+rc is tagged, the PRM branches vX
+
+4. Starting with the pins made by RM on Coq's CI all packages part of the
+   platform (or its core) are pinned (in accordance with upstreams, which are
+   notified about the ongoing process). Most, if not all, packages are in Coq's
+   CI so there should be no surprises, otherwise issues are reported to Coq
+   devs. When done a X+beta tag is done and packages made available to users.
+   A docker image with the entire platform prebuilt should be built. This
+   should take 20 days.
+5. As users pick up the platform and find severe bugs in Coq, the platform picks
+   up point releases of Coq containing hotfixes and eventually extends packages
+   beyond the core set.
+
+## Synchronization points
+
+- The end of (1) starts the release cycle of the platform.
+- The end of (2) and (4) don't need to be done at the same time, but if they
+  are then the new release cycle coincides with the old one (up to the
+  renaming of X.0 into X+beta).
```

Comment:
Hi all, I've just read the CEP once and after this first reading, it seems the least clear passage is this item.
In particular, I believe one could precise:
* that even if (2) (coq vX.0) and (4) (platform vX+beta) need not to be done at the same time, maybe there should anyway be a constraint such as time(4) ≥ time(2) ?
* and rephrase or elaborate on this sentence: "the new release cycle coincides with the old one (up to the renaming of X.0 into X+beta)" which does not seem clear to me… (w.r.t. "the old one" and "up to the renaming")